runtime/doc/syntax.txt

```diff
@@ -6118,5 +6118,28 @@ literal text specify the size of that text (in bytes):
 		many places.
 "<\@1<=span"	Matches the same, but only tries one byte before "span".
 
+*astro.txt* Astro language indentation and syntax support in Vim.
```

Comment:
I could be wrong of course, but I think the section "5. Syntax file remarks" in the "syntax.txt" file was meant [here](https://github.com/vim/vim/pull/14561#discussion_r1573095191).
There is a specific description format for syntax files and indentation of certain file types.